### ISSUE TITLE
sops became slow all of a sudden

### ISSUE THREAD

**Issue by dfioravanti** [mozilla/sops#789]
Hi, first thanks a lot for this amazing project. Today SOPS all of a sudden became really slow in encrypting anything. I get two deprecation warnings 
```
[PGP]    WARN[0000] Deprecation Warning: GPG key fetching from a keyserver witihin sops will be removed in a future version of sops. See https://github.com/mozilla/sops/issues/727 for more information. 
[PGP]    WARN[0030] Deprecation Warning: GPG key fetching from a keyserver witihin sops will be removed in a future version of sops. See https://github.com/mozilla/sops/issues/727 for more information.
```
and then it stalls for a while and then it encrypts. Any idea what could be going on? Until yesterday everything was fine. I reproduced this problem on multiple PC.

**Comment by dfioravanti** [comment]
If I had to guess it feels like a timeout since https://keys.openpgp.org seams to be down. Is it possible to disable the key fetching ?

**Comment by dominykas** [comment]
There doesn't seem to be a way to disable this, I think: https://github.com/mozilla/sops/blob/38b25bd449619e1d6da20e637702f7c73203aa44/pgp/keysource.go#L90-L93

**Comment by dfioravanti** [comment]
I am a bit confused, the documentation says 

> By default, ``sops`` uses the key server ``keys.openpgp.org`` to retrieve the GPG keys that are not present in the local keyring.

but all my keys are present in the keyring. Why is it waiting for openpgp?

**Comment by dominykas** [comment]
Seems the doc is incorrect - it tries to fetch from the keyserver first, if I'm reading the code correctly.

**Comment by supermarin** [comment]
Nope, loads from the keyring first: https://github.com/mozilla/sops/blob/38b25bd449619e1d6da20e637702f7c73203aa44/pgp/keysource.go#L109

Can confirm that it's pretty slow here as well.
